src/features/island/fruit/FruitPatch.tsx

```diff
@@ -26,6 +25,46 @@ export const isReadyToHarvest = (
   );
 };
 
+import appleTree from "assets/fruit/apple/apple_tree.png";
```

Comment:
Can we please remove this file from the PR?

Reply:
aight